packages/launch/src/focused-launch/success/index.tsx

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable wpcalypso/jsx-classname-namespace */
+
+/**
+ * External dependencies
+ */
+import React, { useEffect } from 'react';
+import { __ } from '@wordpress/i18n';
+import { Title } from '@automattic/onboarding';
+import { Icon, wordpress } from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import { useFocusedLaunchModal, useSiteDomains } from '../../hooks';
+
+import './style.scss';
+
+const Success: React.FunctionComponent = () => {
+	const { siteSubdomain } = useSiteDomains();
+	const { unsetModalDismissible, closeFocusedLaunch } = useFocusedLaunchModal();
+
+	// When in the Success view, the user can't dismiss the modal anymore
+	useEffect( () => {
+		unsetModalDismissible();
+	}, [ unsetModalDismissible ] );
+
+	return (
+		<div>
+			<Title>{ __( 'Hooray!', __i18n_text_domain__ ) }</Title>
+			<p>
+				{ __(
+					"Congratulations, your website is now live. We're excited to watch you grow with WordPress.",
+					__i18n_text_domain__
+				) }
+			</p>
+
+			{ /* @TODO: this will work only when the modal in in the block editor. */ }
+			<button onClick={ closeFocusedLaunch }>
+				{ __( 'Continue editing', __i18n_text_domain__ ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **33** times:
`translate( 'Continue Editing' )` *ES Score: 11*